src/System.Runtime.Extensions/tests/System/BitConverterArray.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public class BitConverterArray : BitConverterBase
+    {
+        public override void ConvertFromBool(bool boolean, int expected)
+        {
+            Assert.Equal(expected, BitConverter.GetBytes(boolean)[0]);
+        }
+
+        public override void ConvertFromShort(short num, byte[] byteArr)
```

Comment:
Nit: instead of naming these args `byteArr`, consider using `expected` or `expectedByteArr`, so that it's clear it's the expect output rather than some other input.